etcdctl/ctlv3/command/ep_command.go

```diff
@@ -44,6 +48,9 @@ func newEpHealthCommand() *cobra.Command {
 		Short: "Checks the healthiness of endpoints specified in `--endpoints` flag",
 		Run:   epHealthCommandFunc,
 	}
+
+	cmd.Flags().StringVar(&healthCheckKey, "health-check-key", "health", "A key used as a parameter of get request")
```

Comment:
The key used to perform the health check. Makes sure the user can access the key.